pallets/creditcoin/src/tests.rs

```diff
@@ -1471,8 +1485,11 @@ fn add_authority_works_for_root() {
 
 		assert_ok!(Creditcoin::add_authority(crate::mock::Origin::from(root), acct.clone(),));
 
-		let value = Authorities::<Test>::take(acct);
-		assert_eq!(value, Some(()))
+		let value = Authorities::<Test>::get(acct.clone());
+		assert_eq!(value, Some(()));
+
+		let via_getter = crate::Pallet::<Test>::authorities(acct);
+		assert_eq!(via_getter, value);
```

Comment:
I think we should default to using the getters inside tests instead of the storage types directly.

An off-topic question: shouldn't the getter functions be public but the storage types private or pub(crate) perhaps?

Reply:
> I think we should default to using the getters inside tests instead of the storage types directly.

Noted.

> An off-topic question: shouldn't the getter functions be public but the storage types private or pub(crate) perhaps?

I think this should be treated on a per case basis. I like to have everything private in the beginning and increasing visibility depending on the use case.